### PR TITLE
Danger, primary, warning, success alert css done 

### PR DESCRIPTION
**Pull request by rahul0396**
Just make Alert for success, danger, warning, info 
Screenshot (118)
Screenshot (119)

### CODE REVIEW

File: alert.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.alert {
+  border-radius: 0.5rem;
+  font-weight: bold;
+  margin: 1rem 0;
+  max-width: 25rem;
```
Comment (omjadhav85) [inline alert.css]: I think for alerts width would be better in percentage like 70% or 80%

**Review by omjadhav85 — COMMENTED** [review]
Just added some comments about the width. Else looks good!

**Review by shivampandey0 — COMMENTED** [review]
Good use of variables for colors, variables can be used for `rem` units too

File: alert.css (hunk 1 above)
Comment (rahul0396) [reply alert.css]: doned the changes max-widht: 70%;

**Pull request merged**